tester_app/client/app/bundles/HelloWorld/containers/HelloWorld.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes } from 'react';
+import HelloWorldWidget from '../components/HelloWorldWidget';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import Immutable from 'immutable';
+import * as helloWorldActionCreators from '../actions/helloWorldActionCreators';
+
+function select(state) {
+  // Which part of the Redux global state does our component want to receive as props?
+  return { $$helloWorldStore: state.$$helloWorldStore };
```

Comment:
// Note the use of `$$` to prefix the property name because the value is of type Immutable.js